src/ProgramBlockEditor.scss

```diff
@@ -194,3 +194,7 @@ button.ProgramBlockEditor__program-block--active {
     left: #{-$c2lc-action-panel-width / 2};
     top: #{-$c2lc-action-panel-height * 1.3};
 }
+
+.ProgramBlockEditor__program-sequence-scroll-container.isDragging {
```

Comment:
We're using BEM naming for our classes, please rename to `.ProgramBlockEditor__program-sequence-scroll-container--isDragging`

Reply:
I fixed this one.